src/main/java/de/fearnixx/jeak/JeakBot.java

```diff
@@ -185,6 +186,7 @@ protected void doServiceStartup() {
         initializeService(new UserService());
         initializeService(new TokenService());
         initializeService(new RestControllerService());
+        initializeService(new VoiceConnectionService());
```

Comment:
As already discussed, there should be a feature flag for the time the service is experimental. If not enabled, a non-active dummy should be used.